src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
```

Comment:
The parameter `days` could be more descriptive. Consider renaming it to `numberOfDays` to make it clearer.